platform-configs/protectli-vp2420.robot

```diff
@@ -9,9 +9,9 @@ ${FLASHING_METHOD}=                 internal
 # eMMC driver support
 ${E_MMC_NAME}=                      8GTF4R
 ${DMIDECODE_SERIAL_NUMBER}=         N/A
-${DMIDECODE_FIRMWARE_VERSION}=      Dasharo (coreboot+UEFI) v1.2.0
+${DMIDECODE_FIRMWARE_VERSION}=      Dasharo (coreboot+UEFI) v1.2.1-rc1
 ${DMIDECODE_PRODUCT_NAME}=          VP2420
-${DMIDECODE_RELEASE_DATE}=          10/12/2023
+${DMIDECODE_RELEASE_DATE}=          18/12/2024
```

Comment:
```
DMI004.001 Verify the firmware release date :: Check whether the f... | FAIL |
'# dmidecode 3.3
Getting SMBIOS data from sysfs.
SMBIOS 3.3.0 present.

Handle 0x0000, DMI type 0, 26 bytes
BIOS Information
        Vendor: 3mdeb
        Version: Dasharo (coreboot+UEFI) v1.2.1-rc1
        Release Date: 12/18/2024
        ROM Size: 16 MB
        Characteristics:
                PCI is supported
                PC Card (PCMCIA) is supported
                BIOS is upgradeable
                Selectable boot is supported
                ACPI is supported
                Targeted content distribution is supported
                UEFI is supported
        BIOS Revision: 1.2
        Firmware Revision: 0.0' does not contain 'Release Date: 18/12/2024'

```
it looks like you added date in european format and the bios info has this date in american

Reply:
https://github.com/Dasharo/open-source-firmware-validation/pull/643/commits/96b0c24f5102d0e00927719750f2729019c791f3

Reply:
thanks